static-site/src/components/splash/userGroups.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { Fragment } from "react";
+import ScrollableAnchor from "react-scrollable-anchor";
+import * as Styles from "./styles";
+import Cards from "../Cards";
+import { HugeSpacer, FlexCenter } from "../../layouts/generalComponents";
+import { theme } from "../../layouts/theme";
+
+const UserGroups = (props) => {
+
+  const colors = [...theme.titleColors];
+
+  const groupCards = props.user.groups.map((group) => {
+    const groupColor = colors[0];
+    colors.push(colors.shift());
+
+    return (
+      {
+        img: "empty.png",
+        url: `/groups/${group}`,
+        title: group,
+        color: groupColor
+      }
+    );
+  });
+
+  return (
+    <Fragment>
+      <ScrollableAnchor id={'groups'}>
+        <Styles.H1> {props.user.username}&apos;s Groups</Styles.H1>
```

Comment:
Please change this heading to be "Private nextstrain groups" and the subtext to be "Nextstrain groups are collections of datasets with controlled access. You (userName) have access to the following groups:". 

In the future we need to add documentation around groups and link to that here, but that's not part of this PR. I think it'd be great to add a lock icon to further indicate that they are not public, but that can be a separate PR.